ibc/test_setup.go

```diff
@@ -143,23 +143,23 @@ func StartChainsAndRelayerFromFactory(
 	}
 
 	if err := srcChain.Initialize(testName, home, pool, networkID); err != nil {
-		return errResponse(err)
+		return errResponse(fmt.Errorf("failed to initialize source chain: %w", err))
```

Comment:
Wrapping these errors was very helpful in debugging some self-inflicted test failures during development.